Resources/doc/index.rst

```diff
@@ -136,6 +136,12 @@ Now, you can start working with migrations by generating a new blank migration
 class. Later, you'll learn how Doctrine can generate migrations automatically
 for you.
 
+.. code-block:: terminal
+
+    $ php bin/console make:migration
```

Comment:
This is not part of the DoctrineMigrationsBundle code and shouldn't be part of the documentation for the bundle.